src/main/java/org/mdz/search/solrocr/util/ContextBreakIterator.java

```diff
@@ -60,7 +66,10 @@ public int previous() {
 
   @Override
   public int following(int offset) {
-    int limit = this.limitIter.following(offset);
+    int limit = Integer.MAX_VALUE;
+    if (limitIter != null) {
```

Comment:
Vgl. Zeilen 42 und 46: wäre gut, wenn Referenzierung auf `limitIter` einheitlich wäre (also entweder `this.limitIter` oder nur `limitIter`) :)